pkg/rpc/server.go

```diff
@@ -225,6 +227,32 @@ Methods:
 		} else {
 			results = "Invalid public account address"
 		}
+	case "getrawtransaction":
+		param0, err := reqParams.ValueWithType(0, "string")
+		if err != nil {
+			resultsErr = err
+		} else if txHash, err := util.Uint256DecodeString(param0.StringVal); err != nil {
+			err = errors.Wrapf(err, "param at index 0, (%s), could not be decode to Uint256", param0.StringVal)
+			resultsErr = NewInvalidParamsError(err.Error(), err)
+		} else if tx, height, err := s.chain.GetTransaction(txHash); err != nil {
+			err = errors.Wrapf(err, "Invalid transaction hash: %s", txHash)
+			resultsErr = NewInvalidParamsError(err.Error(), err)
+		} else if len(reqParams) >= 2 {
+			_header := s.chain.GetHeaderHash(int(height))
+			header, err := s.chain.GetHeader(_header)
+
+			param1, exists := reqParams.ValueAtAndType(1, "number")
```

Comment:
according to C# code:
- False - for empty / 0 / "0" / false / " false"
- True - otherwise